draft-ietf-quic-transport.md

```diff
@@ -2263,7 +2263,7 @@ following layout:
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|0|1|                   Random Bytes (166..)                  ...
+|0|1|                   Random Bits (190..)                  ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
Shouldn't this be 182?

The minimum sum of length(pn) + length(payload) is 2 in the current draft. That changes to 4 in the PR. Therefore the delta is 2*8, not 3*8.

Reply:
And I very carefully calculated this, having first reached this conclusion (182), then recalculated and got 190.  Thanks.